pkg/helmdeploy/write.go

```diff
@@ -0,0 +1,212 @@
+package helmdeploy
```

Comment:
How about we move the `helmdeploy`, `kustomize`, `diff` packages into a single package and name it `app-archive`?